### PR TITLE
improvements to how-to aws psql prisma guide

### PR DESCRIPTION
**Pull request by depombo**
For easier review https://github.com/alantech/alan/blob/psql/anycloud/docs/how-to/aws-postgres.md

### CODE REVIEW

**Review by aguillenv — APPROVED** [review]

File: anycloud/docs/how-to/aws-postgres.md [hunk 1]
```diff
@@ -65,15 +70,75 @@ RUN npm run build
...
 
-5) Use the AnyCloud CLI to create an `anycloud.json` file in the project directory and define a `Deploy Config`. You will need to pick a name or alias for the `Deploy Config`. The default value will be `staging`. You will also need to associate `Credentials` to this `Deploy Config`.
+8) Change HTTP server port in `src/index.ts` from `3000` to `8088` which is what what AnyCloud expects.
+
+```
+...
+
+const server = app.listen(8088, () =>
+  console.log(`
+🚀 Server ready at: http://localhost:8088
```
Comment (cdmistman) [inline anycloud/docs/how-to/aws-postgres.md:88]: this looks weird just because it's not closed out...

**Review by dfellis — APPROVED** [review]
This looks complete and correct to me now.

I do also think this is also a wake-up call of sorts. There's a *lot* of extra steps due to RDS/Postgres being involved. It doesn't matter that we let people use their credits unlike some of the competition like Railway if their instructions are 1/4th the size of this :/

We have a good MVP, though, and for some the extra work with RDS isn't "that hard" because they've done it before, and the savings with our approach versus Kubernetes is worth it, but I think we need to PM up a roadmap for AnyCloud that tackles this developer experience deficit exposed by this blogpost.

**Comment by depombo** [comment]
> This looks complete and correct to me now.
> 
> I do also think this is also a wake-up call of sorts. There's a _lot_ of extra steps due to RDS/Postgres being involved. It doesn't matter that we let people use their credits unlike some of the competition like Railway if their instructions are 1/4th the size of this :/
> 
> We have a good MVP, though, and for some the extra work with RDS isn't "that hard" because they've done it before, and the savings with our approach versus Kubernetes is worth it, but I think we need to PM up a roadmap for AnyCloud that tackles this developer experience deficit exposed by this blogpost.

100% agree and had the same thought myself. I'm still thinking about a proposal, but already on it. I see it as a spectrum of possibilities going from least involved to most involved:
- This guide in it's current state
- Making a script that users can run which spins up the Postgres Instance and connects it to one of their anycloud apps
- Offering something like ["add-ons" as Heroku does](https://elements.heroku.com/addons) so that the user can add Postgres in each cloud provider through the CLI and we move the code from the script described above into the `deploy` service
- We have also discussed offering a hosted Postgres ourselves. We spin up a Postgres in every VM and have a write main and multiple read replicas that can auto-promote if the main goes down. This is the most involved, but we can eventually offer a data migration tool to really squash vendor portability without having to use our datastore.

File: anycloud/docs/how-to/aws-postgres.md [hunk 2]
```diff
@@ -65,15 +70,78 @@ RUN npm run build
...
+⭐️ See sample requests: http://pris.ly/e/ts/rest-express#3-using-the-rest-api`,
+  ),
+)
+
+...
+```
+
+9) Make sure everything works locally
+
+```javascript
```
Comment (cdmistman) [inline anycloud/docs/how-to/aws-postgres.md:98]: should this maybe be bash instead?

Comment (depombo) [reply anycloud/docs/how-to/aws-postgres.md:98]: yes!

**Pull request merged**